php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1499,6 +1499,35 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         return null;
     }
 
+    /**
+     * Determines what the data entry completion progress percentage is
```

Comment:
What is the point of this function being added?

Reply:
for new candidate profile page (completion column) 
<img width="698" alt="Screenshot 2020-01-28 12 52 05" src="https://user-images.githubusercontent.com/17415878/73290783-0834ca80-41cd-11ea-8be2-30673ff134b1.png">